paasta_tools/cli/cmds/spark_run.py

```diff
@@ -494,7 +495,9 @@ def get_docker_run_cmd(
     return cmd
 
 
-def get_docker_image(args, instance_config):
+def get_docker_image(
```

Comment:
It would be better if we get repo digest here.
Because we don't know what kind of url user might provide in the `args.image` or from from service config.
The principal here should be 
`The image we get | pull | build and run in driver MUST be exactly the same with the image we run in executor.`
For this to be true, we must use the repo digest for driver and for executor.